assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -601,6 +613,36 @@ const baseSelectors = {
 		return module.dependants.map( ( dependantSlug ) => modules[ dependantSlug ]?.name || dependantSlug );
 	} ),
 
+	/**
+	 * Gets module store name by slug.
+	 *
+	 * Returns the store name if preset or null if there is no store name for this module.
+	 * Returns `undefined` if state is still loading or if said module doesn't exist.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {string} slug Module slug.
+	 * @return {(string|null|undefined)} `string` of the store name if a name has been set for this module.
+	 * 									 `null` if no store name was set.
+	 * 									 `undefined` if state is still loading.
+	 */
+	getModuleStoreName: createRegistrySelector( ( select ) => ( state, slug ) => {
+		const module = select( STORE_NAME ).getModule( slug );
+
+		// Return `undefined` if module with this slug isn't loaded yet.
+		if ( module === undefined ) {
+			return undefined;
+		}
+
+		// Return null if no store name was set
+		if ( module === null ) {
+			return null;
+		}
+
+		// Return the store name.
```

Comment:
This is a bit of a nitpick but let's not add comments that don't add additional context. We don't need to document every line 😄 

```suggestion
```